PwshSpectreConsole.Tests/writing/Write-SpectreCalender.tests.ps1

```diff
@@ -68,5 +61,29 @@ Describe "Write-SpectreCalendar" {
             }
             Assert-MockCalled -CommandName "Write-AnsiConsole" -Times 1 -Exactly
         }
+
+        It "Should match the snapshot" {
+            Mock Write-AnsiConsole {
+                $testConsole.Write($RenderableObject)
+            }
+            $events = @{
+                '2022-03-10' = 'Event 1'
+                '2022-03-20' = 'Event 2'
+            }
+            $culture = Get-Culture -Name "en-US"
+            Write-SpectreCalendar -Date 2024-07-01 -Culture $culture -Events $events -Border "Rounded" -Color "SpringGreen3"

```

Comment:
-Culture is [cultureinfo] type,
so you can just pass 'en-us' to it dont need to do Get-Culture.